pkg/compiler/test/policy_compiler_test.go

```diff
@@ -61,6 +61,8 @@ components:
 			properties:
 				id:
 					type: string
+				tag:
+					type: string
```

Comment:
we would like tags implemented in this other way:
https://github.com/infobloxopen/seal/issues/39